package.json

```
@@ -22,12 +22,15 @@
   },
   "homepage": "https://github.com/mcollina/loopbench#readme",
   "devDependencies": {
-    "pre-commit": "^1.1.2",
-    "standard": "^5.3.1",
-    "tap-spec": "^4.1.0",
-    "tape": "^4.2.2"
+    "pre-commit": "^1.2.2",
+    "standard": "^16.0.4",
+    "tap-spec": "^5.0.0",
+    "tape": "^5.5.2"
   },
   "dependencies": {
-    "xtend": "^4.0.1"
+    "xtend": "^4.0.2"
+  },
+  "engines": {
+    "node": ">=10.7.0"
```

```suggestion
    "node": ">=12.0.0"
```

Let's mark as v12 needed, v10 is out of maintenance. I'll bump the major.